sstables/sstables.cc

```diff
@@ -1391,6 +1391,12 @@ future<> sstable::update_info_for_opened_data(sstable_open_config cfg) {
     if (cfg.load_first_and_last_position_metadata) {
         co_await load_first_and_last_position_in_partition();
     }
+    if (auto opt_gen = _components->scylla_metadata->get_optional_sstable_generation()) {
+        _original_sstable_generation = generation_type(*opt_gen);
+    }
+    if (cfg.load_first_and_last_position_metadata) {
+        co_await load_first_and_last_position_in_partition();
+    }
```

Comment:
Why is this duplicated here? I already see this snippet above.

Reply:
weird. maybe cut-and-paste pilot error.
will remove the redundant code.